vue/components/ui/atoms/avatar/avatar.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="avatar" v-bind:class="[{ notify: notify }, size]">
+        <div class="image-container" v-if="hasImg">
+            <img class="image" v-bind:src="imgUrl" alt="avatar" />
+            <slot name="dot">
+                <div class="dot" />
+            </slot>
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.avatar {
+    display: inline-block;
+    font-size: 0px;
+    white-space: nowrap;
+}
+
+.avatar .image-container {
+    display: inline-block;
+    font-size: 0px;

```

Comment:
Is this needed?

Reply:
Yes, so that it has exact same size as the image. HTML will have. space in between `img` and `slot` adding a white space which has it's own size, increasing the whith. Unless `font-size: 0px;`.